docs/development/howto.rst

```diff
@@ -761,19 +761,18 @@ Gammapy has a class for generic n-dimensional data arrays,
 should use this class. The goal is to reuse code for interpolation
 and have an coherent I/O interface, mainly in `~gammapy.irf`.
 
-A usage example can be found in :gp-extra-notebooks:``nddata_demo``.
+A usage example can be found in :gp-extra-notebook:`nddata_demo`.
 
 Also, consult :ref:`interpolation-extrapolation` if you are not sure how to
 setup your interpolator.
 
-
 Sphinx docs build
 -----------------
 
 Generating the HTML docs for Gammapy is straight-forward::
 
-    python setup.py build_docs
-    open docs/_build/html/index.html
+    make docs-all
+    make doc-show
```

Comment:
Maybe rename the make target `doc-show` to `docs-show`, for consistency with `docs-all`?